src/components/pages/Navbar/NavbarFeatures/LoginButton.js

```diff
@@ -3,12 +3,20 @@ import { useAuth0 } from '@auth0/auth0-react';
 import { NavBtnLink } from '../../NavBarLanding/NavBarStyle';
 
 function LoginButton() {
-  const { loginWithRedirect } = useAuth0();
-
+  // const { loginWithRedirect } = useAuth0();
+  function loginRoute() {
+    window.location = '/login';
+  }
   return (
```

Comment:
Left this code commented out since it will most likely be used once Auth0 has been fully implemented